src/System.Text.Json/src/System/Text/Json/JsonException.cs

```diff
@@ -48,6 +48,14 @@ public JsonException(string message, string path, long? lineNumber, long? bytePo
             Path = path;
         }
 
+        /// <summary>
+        /// Creates a new exception object with serialized data.
+        /// </summary>
+        /// <param name="info">The <see cref="SerializationInfo"/> that holds the serialized object data about the exception being thrown.</param>
+        /// <param name="context">The <see cref="StreamingContext"/> that contains contextual information about the source or destination.</param>
+        /// <exception cref="ArgumentNullException">
+        /// Thrown when <paramref name="info"/> is <see langword="null" />.
```

Comment:
An exception thrown by an exception. Should they be named "inceptions" instead? :)